canonical-web-and-design/jaas-dashboard
# Use Maas <MainTable> component instead of <TableSet>

## Done

Use Maas `<MainTable>` component instead of `<TableSet>`

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify table is sortable by clicking column headers
